plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/PluginJenkinsRule.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.grandhackathon.deployment.sphere.plugin;
+
+import hudson.Functions;
+
+import java.io.File;
+import java.net.URLConnection;
+
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class PluginJenkinsRule extends JenkinsRule{
```

Comment:
Fix integration testing on Windows environment.
Details https://wiki.jenkins-ci.org/display/JENKINS/Unit+Test+on+Windows - "Jenkins core caches and keep file handles"